src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -197,9 +197,16 @@ def base64_encoded_certificate_type(string):
     return cert_data
 
 def datetime_type(string):
```

Comment:
You should remove 'datetime_type' and 'datetime_string_type' since you are using the parameterized version.